checkly/resource_check.go

```diff
@@ -280,13 +280,12 @@ func resourceCheck() *schema.Resource {
 						"ssl_certificates": {
 							Type:       schema.TypeSet,
 							Optional:   true,
-							Deprecated: "The property `ssl_certificates` is deprecated and it's ignored by the Checkly Public API. It will be removed in a future version.",
+							Deprecated: "This property is deprecated and it's ignored by the Checkly Public API. It will be removed in a future version.",
```

Comment:
This change sadly makes it ambiguous which argument has been deprecated. Really Terraform should be dealing with this nicer, so I've raised an issue upstream: https://github.com/hashicorp/terraform-plugin-sdk/issues/1094

Reply:
@RealOrangeOne thanks for the feedback, do you think we can make something for now to improve the DX and make it clear?

Reply:
I guess the original message was better than the new one for user understanding